server/utils/userUtils.test.ts

```diff
@@ -21,6 +26,11 @@ describe('userUtils', () => {
         const expected = [sections.referral, sections.newReferral]
         expect(sectionsForUser(['ROLE_POM'])).toEqual(expected)
       })
+      it('should not return the prison dashboard for a Licence CA', () => {
+        config.flags.isPrisonDashboardEnabled = false
+        const expected: string[] = []
+        expect(sectionsForUser(['ROLE_LICENCE_CA'])).toEqual(expected)
+      })
```

Comment:
would it make sense to remove the feature flag before merging this branch?

Reply:
ah that's a good point yeah, will take a look at your PR